Dockerfile-5.6

```diff
@@ -99,6 +104,6 @@ RUN phpenmod drupal-recommended
 
 ENV PHP_DEFAULT_EXTENSIONS calendar ctype curl dom exif fileinfo ftp gd gettext iconv json mcrypt mysqli mysqlnd opcache pdo pdo_mysql phar posix readline shmop simplexml soap sockets sqlite sysvmsg sysvsem sysvshm tokenizer wddx xml xmlreader xmlwriter xsl mbstring zip
 
-HEALTHCHECK CMD ["sh", "-c", "[ -e /tmp/docker-finished-init ]"]
+HEALTHCHECK --interval=1s --timeout=5s CMD goss -g /goss/goss.yaml validate
```

Comment:
To be complete, shouldn't we add a file https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#file test on /tmp/docker-finished-init